### ISSUE TITLE
es6 exports broken

### ISSUE THREAD

**Issue by modosc** [react-ga/react-ga#170]
as of 2.3.x this no longer works:
```
import { OutboundLink } from 'react-ga'
```

it looks like the es6 build is exporting everything under `default`:
```
import * as ReactGA from 'react-ga'
console.log(ReactGA)
{ default:
   { initialize: [Function: initialize],
     ga: [Function: ga],
     set: [Function: set],
     send: [Function: send],
     pageview: [Function: pageview],
     modalview: [Function: modalview],
     timing: [Function: timing],
     event: [Function: event],
     exception: [Function: exception],
     plugin: { require: [Function: require], execute: [Function: execute] },
     outboundLink: [Function: outboundLink],
     OutboundLink:
      { [Function: OutboundLink]
        propTypes: [Object],
        defaultProps: [Object],
        trackLink: [Function: bound outboundLink],
        origTrackLink: [Function] } } }
```

**Comment by SimeonC** [comment]
See conversation in #165.

I'll try get this export method to work as well as the default one as it was supposed to be a minor change not a major change!

**Comment by SimeonC** [comment]
Closing as this is fixed in 2.3.4.
